src/main/java/com/MyApp/budgetControl/domain/category/CategoryRepository.java

```diff
@@ -1,11 +1,12 @@
 package com.MyApp.budgetControl.domain.category;
 
 import java.util.List;
+import java.util.Optional;
 
-interface CategoryRepository {
+public interface CategoryRepository {
```

Comment:
Does it have to be public?

Reply:
Not really, I suppose it was changed during my prototyping of a working orchestrator solution